features/0511-dif-cred-manifest-attach/README.md

```diff
@@ -0,0 +1,292 @@
+    }],
+    "requests~attach": [{
+        "@id": "7cd11894-838a-45c0-a9ec-13e2d9d125a1",
+        "mime-type": "application/json",
+        "data": {
+            "json": {
+                "credential-manifest": {
+                    "issuer": "did:example:123",
+                    "credential": {
+                        "name": "Washington State Class A Commercial Driver License",
+                        "schema": "ipfs:QmPXME1oRtoT627YKaDPDQ3PwA8tdP9rWuAAweLzqSwAWT"
+                    }
+                },
+                "presentation-submission": {
+                    "@context": [
+                        "https://www.w3.org/2018/credentials/v1",
+                        "https://identity.foundation/presentation-exchange/submission/v1"
+                    ],
+                    "type": [
+                        "VerifiablePresentation",
+                        "PresentationSubmission"
+                    ],
+                    "presentation_submission": {
+                        "descriptor_map": [{
+                            "id": "citizenship_input",
+                            "path": "$.verifiableCredential.[0]"
+                        }]
+                    },
+                    "verifiableCredential": [{
+                        "@context": "https://www.w3.org/2018/credentials/v1",
+                        "id": "https://us.gov/claims/Passport/723c62ab-f2f0-4976-9ec1-39992e20c9b1",
+                        "type": ["USPassport"],
+                        "issuer": "did:foo:123",
+                        "issuanceDate": "2010-01-01T19:73:24Z",
+                        "credentialSubject": {
+                            "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+                            "birth_date": "2000-08-14"
+                        },
+                        "proof": {
+                            "type": "EcdsaSecp256k1VerificationKey2019",
+                            "created": "2017-06-18T21:19:10Z",
+                            "proofPurpose": "assertionMethod",
+                            "verificationMethod": "https://example.edu/issuers/keys/1",
+                            "jws": "..."
+                        }
+                    }],
+                    "proof": {
+                        "type": "RsaSignature2018",
+                        "created": "2018-09-14T21:19:10Z",
+                        "proofPurpose": "authentication",
+                        "verificationMethod": "did:example:ebfeb1f712ebc6f1c276e12ec21#keys-1",
+                        "challenge": "1f44d55f-f161-4938-a659-f8026467f126",
+                        "domain": "us.gov/DriverLicense",
+                        "jws": "..."
+                    }
+                }
+            }
+        }
+    }]
+}
+```
+
+### `issue-credential` attachment format
+
+This specification does not register any format identifier for the `issue-credential` message. The Issuer SHOULD set the `format` to the value
```

Comment:
I don't know that much about the DIF document, but I don't understand why there would not be a format string for this.